adserver/templates/adserver/reports/advertiser-per-ad.html

```diff
@@ -0,0 +1,37 @@
+{% extends "adserver/reports/advertiser.html" %}
+{% load humanize %}
+{% load i18n %}
+{% load metabase %}
+
+
+{% block title %}{% trans 'Advertiser Per Ad Per Day Report' %} - {{ advertiser }}{% endblock %}
+
+
+{% block heading %}
+{% blocktrans %}Advertiser Per Ad Per Day Report for {{ advertiser }}{% endblocktrans %}
+{% endblock heading %}
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item active">{% trans 'Per Ad Per Day Report' %}</li>
+{% endblock breadcrumbs %}
+
+
+{% block explainer %}
+<section class="mb-5">
+  <h3>{% trans 'About this report' %}</h3>
+  <p>{% trans 'This report shows a table of data with one entry for each ad for every day that ad is shown. The data can be exported to CSV/Excel for further analysis with the icon at the bottom right.' %}</p>
```

Comment:
Might want to make the export info a bit more prominent? Perhaps bolded or something?

Reply:
Done